_includes/footer.html

```diff
@@ -47,10 +47,8 @@ <h2 class="footer-heading">{{ site.title }}</h2>
                    <h4>Contact us</h4>
                    <ul>
                         <li>5th Avenue, New York - United States</li>
-                        <li>Phone: +10 724 1234 567 | Fax: +10 724 1234 567 </li>
+                        <li>Phone: +10 724 1234 567</li>
```

Comment:
This should be a variable like `{{ site.phone }}` and there should be a `phone: 000 000 0000` in `_config.yml`